ui/frontend/build_src/src/components/organisms/displayPanel/index.tsx

```diff
@@ -1,18 +1,25 @@
+/* eslint-disable @typescript-eslint/strict-boolean-expressions */
```

Comment:
commented out a lot of code here, this used to be responsible for loading and displaying, now it will do much less.